libraries/iio/iio.c

```diff
@@ -0,0 +1,843 @@
+/***************************************************************************//**
```

Comment:
do we want to have duplicate files?
https://github.com/analogdevicesinc/no-OS/blob/master/iio/iio.c

Reply:
yes, the motivation is in the description